config/routes.rb

```diff
@@ -245,6 +245,24 @@
         get :check
       end
     end
+
+    resources :messages, only: [], path: :meddelande do
+      resources :message_comments, path: :kommentarer, only: [:create, :destroy]
+    end
+
+    resources :groups, path: :grupper, except: [:new, :create, :destroy] do
+      resources :messages, only: [:new, :create, :destroy, :index], path: :meddelande do
+        resources :message_comments, path: :kommentarer, only: [:create, :destroy]
```

Comment:
Do we need both `grupper/meddelanden/kommentarer` and `/meddelanden/kommentarer`?